juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
402
/home/jenkins/.jenkins/workspace/iceberg/jfit-rules_validate/rules/juniper_official/Solutions/Mpls-blackhole-detection/jnhexceptionpkt.yml
  8:1       error    trailing spaces  (trailing-spaces)
  12:1      error    trailing spaces  (trailing-spaces)
  17:1      error    trailing spaces  (trailing-spaces)

Reply:
Addressed and resolved